fedimint-client/Cargo.toml

```diff
@@ -17,7 +17,6 @@ normal = ["aquamarine"]
 rustc-args = ["--cfg", "tokio_unstable"]
 
 [features]
-default = ["tor"]
```

Comment:
Probably some other crates (like `fedimint-cli`) will need to enable it when their feature is set. This was probably incorrectly handled in the first place.

Reply:
The features need to cascade from outter-most to the innermost.

Reply:
Setting it as defaiult in `fedimint-cli` and not default here is the correct setup though right?

Reply:
Yes. But fedimint-cli needs to have:

```
[features]
tor = [ "fedimint-cli/tor" ]
```

too, so when it's enabled on `fedimint-cli`, it will also get enabled on `fedimint-client`.